### ISSUE TITLE
net8.0 not compatible makes testing difficult

### ISSUE THREAD

**Issue by hpathanjaly** [vitalii-vov/Maui.PDFView#7]
I'm pretty new to MAUI so I might have missed something here but since the PdfView is not compatible with net8.0 on its own and xUnit's target framework is net8.0, adding my project as a dependency to the test project led to a bunch of errors wherever the pdf view was mentioned. I had to add preprocessor directives to check whether the program was running on windows, mac, ios, or android and then include the pdf view. this also forced me to add the pdf view element from the code  behind. Was there a better way to do this or did i miss something? I have attached images of my workaround
![image](https://github.com/user-attachments/assets/ce3d85bc-95b1-4af1-bf33-d85b6093a9b5)
![image](https://github.com/user-attachments/assets/ccc69e81-5acb-44b4-a731-f709a236e0cd)

**Comment by vitalii-vov** [comment]
Hello,
This library is written for `.net8`.

I don't understand the essence of your question, as you haven't provided error reports and it's not clear where exactly these errors occur.
.Net MAUI is a cross-platform solution, which implies writing cross-platform code whose execution will depend on the platform it is running on.
`PdfView` is a wrapper, implemented via the `IPdfView` interface, over the platform code, so the code that will be called depends on the platform it is running on. You will need to figure out which platform you want to test and configure your project correctly.

You should analyze compiler errors to understand what the problem is on your side.

So far it seems to me that the errors are not library related.

**Comment by nikitaKlymchukGunnebo** [comment]
I have same issue. My test project targets net8.0 in this way: 
`<TargetFramework>net8.0</TargetFramework>`
and have a reference for my maui project, which also tatgets net8.0, it looks like this:  
```
<TargetFrameworks>net8.0-android</TargetFrameworks>
    <TargetFrameworks>$(TargetFrameworks);net8.0</TargetFrameworks>
```
You can have a look on how maui community toolkit makes taragetting: https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

You will see that they have $(NetVersion) which will be transformed to net8.0 in our case:
`<TargetFrameworks>$(NetVersion);$(NetVersion)-android;$(NetVersion)-ios;$(NetVersion)-maccatalyst</TargetFrameworks>`

So basically, you need to change csproj to make TargetFrameworks look like this:
`<TargetFrameworks>net8.0;net8.0-android;net8.0-ios;net8.0-maccatalyst</TargetFrameworks>`

And fix related issues that might appear

**Comment by vitalii-vov** [comment]
@hpathanjaly @nikitaKlymchukGunnebo 
Please check version 1.0.5

**Comment by nikitaKlymchukGunnebo** [comment]
1.0.5 fixed this issue

**Comment by hpathanjaly** [comment]
![image](https://github.com/user-attachments/assets/e4012881-d550-4c07-885f-608a61b3ed70)
Updated and still showing this wherever I use the package. Am i doing something wrong?

**Comment by hpathanjaly** [comment]
Just fully uninstalling and reinstalling the package seems to have worked. Ill close the issue after a little more testing
